README.md

```diff
@@ -39,41 +39,62 @@ we would like things to flow.
 ### Requirements
 
 * [Apache Maven 3.3.3+](https://maven.apache.org/install.html)
-* [Java 8+](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+* OpenJDK 8
 * Network access to https://repository.sonatype.org/content/groups/sonatype-public-grid
 
 Also, there is a good amount of information available at [Bundle Development](https://help.sonatype.com/display/NXRM3/Bundle+Development#BundleDevelopment-BundleDevelopmentOverview)
 
-NOTE: Version 0.0.2+ of Conan is only compatible with Nexus Repo 3.11.0 or greater due to adding Tree View Filtering. If 
-you need to use this plugin with an older version, please use 0.0.1
-
 ### Building
 
 To build the project and generate the bundle use Maven
 
-    mvn clean package -PbuildKar
+    mvn clean install
 
 If everything checks out, the bundle for Conan should be available in the `target` folder
 
-#### Build with Docker
+## Using Conan With Nexus Repository Manager 3
 
-`docker build -t nexus-repository-conan:0.0.6 .`
+[We have detailed instructions on how to get started here!](https://help.sonatype.com/repomanager3/formats/conan-repositories)
 
-#### Run as a Docker container
+## Compatibility with Nexus Repository Manager 3 Versions
 
-`docker run -d -p 8081:8081 --name nexus nexus-repository-conan:0.0.6` 
+The table below outlines what version of Nexus Repository Manager the plugin was built against:
 
-For further information like how to persist volumes check out [the GitHub repo for our official image](https://github.com/sonatype/docker-nexus3).
+| Plugin Version    | Nexus Repository Manager Version |
+|-------------------|----------------------------------|
+| v0.0.1            | <3.11.0                          |
+| v0.0.2+           | 3.11.0+                          |
```

Comment:
Minor/Q: What is the intent of the "+" on v0.0.2+?  v1 should be after 0.0.2, so I'm confused by the +.

NOTE: I'm also not a huge fan of the "3.11.0+" however it seems to be how other formats work too...my preference would have been a range but seems we're beyond that now=)

Reply:
I see the examples below, there may be through 0.0.6.
If that's the intent (0.0.2 up to 1.0.0) I get it, tho I may have put either the full versions (0.0.2 - 0.0.6) or a dash instead of a plus (to indicate through rather than beyond), e.g. "v0.0.2-".
I'll let you decide tho, feel free to resolve if you think it makes sense as is now.

Reply:
fd4cf415e454dca8bf8137e8cb634e25954a8bab Is it better now?

Reply:
Yeah thanks=)